### PR TITLE
Support release candidates cut

### PR DESCRIPTION
**Pull request by gotbahn**
Breaking turnover into release candidates support

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 
-## [1.0.0] - Unreleased
+## [2.0.0] - Unreleased
 
-Init
+Support release candidates cut
+
+## [1.0.0] - 07.10.2021
```
Comment (nitzanashi) [inline CHANGELOG.md:10]: You need to do backmerges...

File: src/index.js [hunk 2]
```diff
@@ -86,110 +83,51 @@ const getNewVersions = (changelogBefore, changelogAfter) => {
...
+            body: item.body,
+            head: candidateBranch,
+            base: releaseBranch,
+        })
+
+        await octokit.rest.issues.addLabels({
+            owner,
+            repo,
+            issue_number: pr.number,
+            labels: ['release', 'needs qa'],
```
Comment (nitzanashi) [inline src/index.js:126]: Ideally, this should come as optional params on the action input.

**Review by nitzanashi — APPROVED** [review]
I like how you improved the process, really nice

File: CHANGELOG.md (hunk 1 above)
Comment (gotbahn) [reply CHANGELOG.md:10]: backmerged

File: src/index.js (hunk 2 above)
Comment (gotbahn) [reply src/index.js:126]: https://github.com/zattoo/cactus/issues/7

**Comment by gotbahn** [comment]
I renamed release candidate branches from `candidate` to `rc`

**Review by nitzanashi — APPROVED** [review]

**Pull request merged**